src/runtime_src/core/common/drv/kds_core.c

```diff
@@ -81,6 +81,78 @@ static ssize_t kds_create_cu_string(struct xrt_cu *xcu,
 	return cu_sz;
 }
 
+ssize_t show_kds_cuctx_stat_raw(struct kds_sched *kds, char *buf, uint32_t domain)
```

Comment:
Please refer #7124 for overcome the 4KB limitation by use binary sysfs node.

Reply:
Fixed